app/models/appointment.rb

```diff
@@ -37,14 +37,8 @@ class Appointment < ApplicationRecord
 
   def self.needing_reminder
     pending
-      .includes(:activities)
-      .where(start_at: Time.zone.now..24.hours.from_now)
-      .where.not(
-        email: '',
-        activities: {
-          type: 'ReminderActivity', created_at: 12.hours.ago..12.hours.from_now
-        }
-      )
+      .where(start_at: Time.zone.now..(Time.zone.now + 48.hours))
```

Comment:
Same as below but with the `Time.zone.now..48.hours.from_now.in_time_zone` perhaps more concise, though I'm not totally sure...